spec/models/manageiq/providers/amazon/aws_refresher_spec_counts.rb

```diff
@@ -147,7 +147,7 @@ def table_counts_from_api
       :custom_attribute              => custom_attributes_count,
       :disk                          => disks_count,
       :firewall_rule                 => firewall_rules_count,
-      :flavor                        => 103,
+      :flavor                        => 114,
```

Comment:
How did we gain 11 flavors?  Was this just the count that was missing since the last update, or are those discontinued ones?

Reply:
For example: https://aws.amazon.com/blogs/aws/amazon-ec2-update-additional-instance-types-nitro-system-and-cpu-options/

Reply:
So you are saying those are new instances since our last update of the yaml file?  I'm more asking because when I diffed the lists before your PR and after, I didn't see 11 new flavors in the yml file.

Reply:
See https://github.com/ManageIQ/manageiq-providers-amazon/pull/449#issuecomment-404568411